op-batcher/batcher/driver.go

```diff
@@ -324,6 +430,54 @@ func (l *BatchSubmitter) loop() {
 	}
 }
 
+func (l *BatchSubmitter) getEconomicDAType(ctx context.Context) (flags.DataAvailabilityType, error) {
+	sCtx, sCancel := context.WithTimeout(ctx, l.Config.NetworkTimeout)
+	defer sCancel()
+	gasPrice, err := l.L1Client.SuggestGasTipCap(sCtx)
+	if err != nil {
+		return "", fmt.Errorf("getEconomicDAType failed to fetch the suggested gas tip cap: %w", err)
+	}
+	calldataCost := big.NewInt(0).Mul(big.NewInt(6*ApproximateGasPerCallDataTx), gasPrice)
```

Comment:
Why is it 6*ApproximateGasPerCallDataTx? What does 6 represent?
Suggest giving 6 a constant name for better understanding.